timed/settings.py

```diff
@@ -61,6 +61,7 @@ def default(default_dev=env.NOTSET, default_prod=env.NOTSET):
     "django.contrib.staticfiles",
     "rest_framework",
     "django_filters",
+    "django_extensions",
```

Comment:
`django-extensions` is a dev dependency (and should be), so this is not gonna work in prod.

You should add a block like this below `INSTALLED_APPS`:

```python
if ENV == "dev":
    INSTALLED_APPS.append("django_extensions")
```

Reply:
Done. Thanks a lot.